setup.cfg

```diff
@@ -6,6 +6,11 @@ extend-ignore =
    E203,
 
 per-file-ignores =
+    src/twisted/internet/defer.py:
+        # redefinition of unused 'DeferredList' from line 1437
```

Comment:
Wouldn't a `  # noqa: F81` comment be best here?

Here we ignore all errors inside the file.
With `  # noqa` we should be able to ignore only that line.